src/alg_utils.jl

```diff
@@ -220,7 +222,7 @@ alg_order(alg::VCABM3) = 3
 alg_order(alg::VCABM4) = 4
 alg_order(alg::VCABM5) = 5
 
-alg_order(alg::VCABM) = 7
+alg_order(alg::VCABM) = 1  #dummy value
```

Comment:
just make it nothing so that way if it is used it'll error. We want to make sure it isn't used

Reply:
making it nothing gives error as this function will be called in starting for calculation of `beta1` and `beta2`